dockerfiles/Dockerfile

```diff
@@ -9,6 +9,7 @@ ENV LC_ALL en_US.UTF-8
 
 RUN apt-get -y update
 RUN apt-get -y install \
+        cargo \
```

Comment:
If cargo is being required here, it probably needs to be  in our build images too. Ref https://github.com/readthedocs/readthedocs-docker-images/pull/136

Reply:
It's not really needed here with the newer version of `orjson`. I removed it.